quipucords/fingerprinter/__init__.py

```diff
@@ -847,13 +838,6 @@ def _process_satellite_fact(source, fact):
                             'infrastructure_type', fingerprint,
                             fact_value=infrastructure_type)
 
-    virtualized_is_guest = bool(
```

Comment:
Convert the values here to an infrastructure_type of `bare metal` or `virtualized`